resources/data.go

```diff
@@ -104,17 +105,16 @@ func AssetDir(name string) ([]string, error) {
 }
 
 type _bintree_t struct {
-	Func func() ([]byte, error)
```

Comment:
this file shouldn't have changed unless we changed the config files. Did we make any changes to default_config.yaml ?

Reply:
if not, could you revert it back ?

Reply:
for some reason running `make format-go` changes this file. I can revert it back.

Reply:
i think it's just formatting changes from `make format-go`